fastjet/internal/delaunay/triangle_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"testing"
+)
+
+func TestTriangleEquals(t *testing.T) {
+	tests := []struct {
+		a, b *Triangle
+		want bool
+	}{
+		{NewTriangle(NewPoint(3, 2), NewPoint(0, 0), NewPoint(5, 0)), NewTriangle(NewPoint(3, 2), NewPoint(0, 0), NewPoint(5, 0)), true},
+		{NewTriangle(NewPoint(3, 2), NewPoint(0, 0), NewPoint(5, 0)), NewTriangle(NewPoint(5, 0), NewPoint(3, 2), NewPoint(0, 0)), true},
+		{NewTriangle(NewPoint(3, 2), NewPoint(0, 0), NewPoint(5, 0)), NewTriangle(NewPoint(3, 1), NewPoint(0, 0), NewPoint(5, 0)), false},
+		{NewTriangle(NewPoint(3, 2), NewPoint(0, 0), NewPoint(5, 0)), NewTriangle(NewPoint(5, 0), NewPoint(3, 1), NewPoint(0, 0)), false},
+	}
+	for _, test := range tests {
+		got := test.a.Equals(test.b)
+		if got != test.want {
+			t.Fatalf("%v == %v,\n got = %v, want = %v", test.a, test.b, got, test.want)
```

Comment:
use `t.Errorf` instead of `t.Fatalf` so the other test cases can run.
also: please add the index `i` of the test-case being run in the printout.